build.gradle

```diff
@@ -111,8 +110,11 @@ import com.sun.jna.Platform as DefaultPlatform
 ext.libsGitSha = "git --git-dir=${rootProject.rootDir}/.git diff --name-only master -- :/libs".execute().text.allWhitespace ?
                  "git --git-dir=${rootProject.rootDir}/.git rev-parse HEAD:libs".execute().text.trim() : null
 
+// Use in-tree libs from the source directory in CI or if the git SHA is unset; otherwise use
```

Comment:
Might be worth using System.getenv('CI') in the `ext.libsGitSha` computation, maybe.

Although, arguably it wouldn't be a bad thing for CI to use downloaded libs if the libs dir hasn't changed... Although, it probably would need a slightly different command to determine that than the above.